flow/scripts/synth.tcl

```diff
@@ -50,6 +50,10 @@ if {[info exist ::env(DFF_LIB_FILE)]} {
 }
 opt
 
+# rename registers to have the verilog register name in its name
+# of the form \regName$_DFF_P_. We should fix yosys to make it the reg name.
+# At least this is predictable. Must happen right after dfflibmap.
+renames -wire
```

Comment:
This does appear to help somewhat with net names.  However it doesn't have anything to do with register names.  Also the command needs to appear before 'opt' as the comment says.